main.scss

```diff
@@ -75,14 +75,7 @@
 		position: relative;
 		border-top: 1px solid;
 		border-bottom: 1px solid;
-
-		// Hack to increase sepcificity to override overflow
-		// set on class used on the same element preventing use of !important.
-		// This Sass selector duplicates the class, creating:
-		// .o-footer__external-link.o-footer__external-link
-		&#{&} {
```

Comment:
What a naughty hack

Reply:
I also don't quite get what it was for... or how it would prevent `!important`...